### PR TITLE
Enable LCP audiobooks

### PR DESCRIPTION
**Pull request by mauricecarrier7**
**What's this do?**
Enables LCP Audiobooks

**Why are we doing this? (w/ Notion link if applicable)**
https://www.notion.so/lyrasis/Get-LCP-for-audiobooks-working-in-the-Palace-iOS-app-a6058f8fa3f243b9ac40902c0cdea0ac

**How should this be tested? / Do these changes have associated tests?**
Attempt to open and listen to an LCP Audiobook

**Dependencies for merging? Releasing to production?**
(https://github.com/ThePalaceProject/ios-audiobooktoolkit/commit/4db05800498629be1c0de697ba1f6bb2779e9d8d)

**Does this include changes that require a new Palace build for QA?**
Yes

**Has the application documentation been updated for these changes?**
N/A

**Did someone actually run this code to verify it works?**
@mauricecarrier7

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -14,3 +14,5 @@ Carthage*
 **/TPPSecrets.swift
 fastlane/report.xml
 fastlane/*_output
+Cartfile
```
Comment (vladimirfedorov) [inline .gitignore:17]: We can't ignore `Cartfile` and `Cartfile.resolved`, and they already are in git

File: Cartfile [hunk 2]
```diff
@@ -5,4 +5,5 @@ github "ThePalaceProject/ios-provider-pdf-renderer" "main"
 github "PureLayout/PureLayout" ~> 3.1.2
 github "stephencelis/SQLite.swift" == 0.12.2
 github "zxingify/zxingify-objc" ~> 3.6
-github "readium/swift-toolkit" "main"
\ No newline at end of file
+github "readium/swift-toolkit" "main"
+binary "https://liblcp.dita.digital/readium/liblcp/ios/carthage/xHwy01rpnw/liblcp.json" ~> 2.0.0
```
Comment (vladimirfedorov) [inline Cartfile:9]: could you remove liblcp link here? we shouldn't have them in public repos

File: Cartfile.resolved [hunk 3]
```diff
@@ -12,4 +12,5 @@ github "readium/swift-toolkit" "d887e5874d5024bca42dc396d869b28350ed6cb9"
 github "scinfu/SwiftSoup" "2.3.3"
 github "stephencelis/SQLite.swift" "0.12.2"
 github "weichsel/ZIPFoundation" "0.9.11"
-github "zxingify/zxingify-objc" "3.6.7"
\ No newline at end of file
+github "zxingify/zxingify-objc" "3.6.7"
+binary "https://liblcp.dita.digital/readium/liblcp/ios/carthage/xHwy01rpnw/liblcp.json" "2.0.0"
```
Comment (vladimirfedorov) [inline Cartfile.resolved:16]: could you remove liblcp link here? we shouldn't have them in public repos

File: Palace.xcodeproj/project.pbxproj [hunk 4]
```diff
@@ -3900,7 +3900,7 @@
 				CLANG_WARN_NON_LITERAL_NULL_CONVERSION = YES_ERROR;
 				CODE_SIGN_ENTITLEMENTS = Palace/SimplyE.entitlements;
 				CODE_SIGN_IDENTITY = "iPhone Developer";
-				CURRENT_PROJECT_VERSION = 44;
+				CURRENT_PROJECT_VERSION = 45;
```
Comment (vladimirfedorov) [inline Palace.xcodeproj/project.pbxproj:3903]: 46 is already in TestFlight, could you change this to 47

**Review by vladimirfedorov — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (mauricecarrier7) [reply .gitignore:17]: nice catch, not sure how this got in

File: Palace.xcodeproj/project.pbxproj [hunk 5]
```diff
@@ -3920,6 +3920,7 @@
 				CLANG_WARN_NON_LITERAL_NULL_CONVERSION = YES_ERROR;
 				CODE_SIGN_ENTITLEMENTS = Palace/SimplyE.entitlements;
 				CODE_SIGN_IDENTITY = "iPhone Developer";
+				CURRENT_PROJECT_VERSION = 47;
 				CURRENT_PROJECT_VERSION = 46;
```
Comment (vladimirfedorov) [inline Palace.xcodeproj/project.pbxproj:3924]: It seems Xcode created two build numbers, it does that sometimes, could you try to change build number to some other and then change back? This usually helps.

**Review by vladimirfedorov — APPROVED** [review]
Other than duplicate CURRENT_PROJECT_VERSION, looks good to me

**Pull request merged**